bldr.env.sample

```diff
@@ -8,6 +8,7 @@
 
 # The endpoint and port for your Postgresql or RDS instance
 # Change only if needed
+# Only change these if using an externally hosted PostgreSQL instead of a local PostgreSQL instance
 export POSTGRES_HOST=localhost
 export POSTGRES_PORT=5432
 
```

Comment:
The comment above lines 5-7 above needs to be changed too right?

Reply:
Those are specific to AWS RDS. Username and password is being managed while setting up resources through portal/cli. I believe if we need to make those variables common then some configuration changes are required.

Reply:
It looks like the config variables get set in install.sh via "source ../bldr.env".  
We might mention in the comment above the RDS section that if we enable the
RDS config then we are required to change the POSTGRES_HOST to the FQDN of the
Azure host.

Reply:
@gangwarshivendra my understanding that even though those `RDS_*` variables are named RDS, they actually would apply to any scenario when using an externally hosted database. If RDS_ENABLED is not set then the install script will deploy a local postgres and builder_api will use the password set in that postgres instance `pwfile`. It seems like a user wanting to use azure postgres or any external postgres would need to enable that and set its user/password to the one created in the external database. Otherwise, how would builder_api know the user/password?

Reply:
@mwrock yes I agree, those steps of setting up hab as username and password were done by user explicitly. Let me confirm this by trying it once.

Reply:
@pozsgaic let me try those once and confirm.

Reply:
@gangwarshivendra  How did you supply the username/password if you are not setting them through the below vars:

#export RDS_ENABLED=false
#export RDS_USER=hab
#export RDS_PASSWORD=hab

I guess the term RDS here refers to AWS RDS for PostgreSQL for short for cloud-hosted service. RDS is also a general term and can refer to some database service hosted on a different cloud.

So in our case, external PostgreSQL can refer to one of the following:
Amazon RDS for PostgreSQL
Azure Database for PostgreSQL
Separate PostgreSQL Instance

Generally, to connect to a database (PostgreSQL here), we need host, port, db-name (default?), username, and password. Sometimes they are supplied through a full URL.

We should consider renaming and regrouping the env variables based on their purpose.

RDS_ENABLED -> INSTALL_DATASTORE or something
#export INSTALL_DATASTORE=false

We can still continue to use the same variables:
#export RDS_USER=hab
#export RDS_PASSWORD=hab

or rename them for avoiding further confusion
#export DB_USER=some_name
#export DB_PASSWORD=some_pass

In any case, these should be close to the HOST and PORT configuration.

Reply:
Thanks Phani, I agree with Phani about renaming those environment variables in terms of username and password. I have made those changes.

@mwrock yes, I have tested that would work for all custom PG installation.